src/io/writer_internal.cpp

```diff
@@ -35,17 +45,14 @@ void WriterInternal::Close()
     auto head14 = file_->GetLasHeader();
     WriteChunkTable();
 
-    // Write COPC Extents VLR.
-    WriteExtents(head14);
-
-    // Always write the wkt for now, since it sets evlr_offset
-    // if (!file_->GetWkt().empty())
-    WriteWkt(head14);
+    // Set hierarchy evlr
+    out_stream_.seekp(0, std::ios::end);
+    head14.evlr_offset = static_cast<int64_t>(out_stream_.tellp());
+    head14.evlr_count += hierarchy_->seen_pages_.size();
```

Comment:
make this a WriteExtents() function

Reply:
It's not what this is though? this just sets the EVLR offset and count for the hierarchy. I could move this code within `WritePageTree` though to clean up a bit if you think its better?

Reply:
sorry, not sure what I was thinking. LGTM